app/views/events/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+  <h1 class="text-center pt-3 pb-2">All Events</h1>
+
+  <div class="d-flex flex-row flex-wrap justify-content-between">
+    <% @events.each do |event| %>
+      <div class="card mb-2" style="width: 15rem;">
+        <div class="card-body">
+          <h5 class="card-title font-weight-bolder"><%= event.name %>
+            <% if event.date > DateTime.now %>
+              <span class="badge badge-success">Upcoming</span>
+            <% else %>
+              <span class="badge badge-danger">Past</span>
+            <% end %></h5>
```

Comment:
- [ ] Create a helper method and render the markup within the helper method instead of doing it straight in the view